src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+ul li {
+  padding: 0;
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  // &__img {
+  //   display: block;
+  //   width: 160px;
+  //   height: 134px;
+  //   background-image: url(../images/imac.jpeg);
+  //   background-position: center;
+  //   background-repeat: no-repeat;
+  //   background-size: contain;
+  //   bac
+  //   margin: 0 3px 40px;
+
+  //   // margin-bottom: 40px;
+  // }
+  &__img {
+    margin: 0 2px 0 3px;
+    img {
+      height: 134px;
+      width: 160px;
+
+      margin-bottom: 36px;
+    }
+  }
+
+  &__title {
+    color: #060b35;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    margin-bottom: 4px;
+  }
+  &__product-code {
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+  &__stars-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: #060b35;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    .price-title {
+      color: #616070;
+      font-size: 12px;
+      line-height: 18px;
+    }
+    .price {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+  &__buy-button {
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-align: center;
+
+    color: #fff;
+    background-color: #00acdc;
+
+    &:hover {
+      background-color: #fff;
+      border: 1px solid #00acdc;
+      color: #00acdc;
+    }
+  }
+}
+
+.stars {
```

Comment:
You need realize all variants of stars (1,2,3..). Please check scss loop and try to do it.